src/save-questions.property.spec.ts

```
@@ -0,0 +1,170 @@
+        appendSpy.mockRestore();
+        removeSpy.mockRestore();
+        urlSpy.mockRestore();
+        revokeSpy.mockRestore();
+      }),
+    );
+  });
+
+  it("logs error when saveFile fails", async () => {
+    await fc.assert(
+      fc.asyncProperty(fc.string(), fc.string(), async (fileName, content) => {
+        const errSpy = vi.spyOn(console, "error").mockImplementation(() => {});
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)